providers/azure/connection/azureinstancesnapshot/snapshot.go

```diff
@@ -20,6 +20,35 @@ const (
 	createdValue   = "cnspec"
 )
 
+type SnapshotCreator struct {
```

Comment:
these are all non-functional changes, moving things around and marking fields as private